tsdb/store.go

```diff
@@ -108,13 +108,15 @@ func (s *Store) Statistics(tags map[string]string) []models.Statistic {
 	for _, database := range databases {
 		sc, err := s.SeriesCardinality(database)
 		if err != nil {
-			s.Logger.Info("Cannot retrieve series cardinality", zap.Error(err))
+			s.Logger.Info("Cannot retrieve series cardinality", zap.Error(err),
```

Comment:
How about `log := s.Logger.With(logger.Database(database))`? I know it's only two places but I'm a fan of injecting the shared structures into the logger; it feels a bit cleaner.